src/fw/drivers/mpu.c

```diff
@@ -60,12 +62,14 @@ static const PermissionMapping s_permission_mappings[] = {
 };
 
 static const uint32_t s_cache_settings[MpuCachePolicyNum] = {
+#if !PLATFORM_OBELIX
```

Comment:
this should be ARMV8 or something rather than PLATFORM_OBELIX -- or it should take my mpu_m33 from the nrf53 era